src/openklant/config/models.py

```diff
@@ -0,0 +1,50 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+
+from solo.models import SingletonModel
+from zgw_consumers.models import Service
+
+from openklant.components.klantinteracties.models import Klantcontact
+from referentielijsten_client.client import ReferentielijstenClient
+
+
+class ReferentielijstenConfig(SingletonModel):
+    enabled = models.BooleanField(default=False)
```

Comment:
```suggestion
    enabled = models.BooleanField(default=False, help_text="Indicates if Referentielijsten is enabled")
```
but i think should be in Dutch, with translations, same for other `help_text`